tests/test-timber-term.php

```diff
@@ -143,7 +149,7 @@ function testGetPostsWithAnyAndCustomTax() {
 
 			$term_id = $this->factory->term->create(array('name' => 'Zong', 'taxonomy' => 'arts'));
 			$posts = $this->factory->post->create_many(5, array('post_type' => 'portfolio' ));
-			$term = new Timber\Term($term_id);
+			$term = Timber::get_term($term_id, 'arts');
```

Comment:
Would this fail without `'arts'` as a second parameter? Or: is this line needed at all? I don’t see `$term` being used in the code.

Reply:
You're right, `$term` isn't used in this case and that line can be safely removed. But the `category` ones fail without the `$taxonomy` param...which raises a larger question about the Term API.

`TermGetter::get_term()` currently takes a second `$taxonomy` param which defaults to `post_tag`, unlike the core API which defaults this param to the empty string. This is going to cause problems. Hence, #2087 .

For now I'm just passing empty strings explicitly everywhere; this question is out of scope here as the immediate goal is to keep the tests passing without instantiating terms directly.